tokio/Cargo.toml

```diff
@@ -47,6 +47,7 @@ io-util = ["memchr", "bytes"]
 # stdin, stdout, stderr
 io-std = []
 macros = ["tokio-macros"]
+metrics = []
```

Comment:
I wonder if it is worth making this a feature flag at all (vs. always on).

Reply:
It's going to be pretty slow on platforms that don't have an `AtomicU64` as we would then go through [this mock][1].

[1]: https://github.com/tokio-rs/tokio/blob/master/tokio/src/loom/std/atomic_u64.rs

Reply:
I don't think it is a big deal. Alternatively, we use AtomicUsize let it wrap and it is up to the receiver of data to handle that.